src/app/feedback/[id]/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import Sidebar from "@/components/sidebar/sidebar";
+import { getBookingInfo, getBoatInfo } from "@/services/notion.service";
+import { useEffect, useState } from "react";
+import router from "next/router";
+import FeedbackForm from "@/components/feedback";
+
+export default function UserFeedbackPage({ params }: { params: { id: string } }) {
+  const [boatInfo, setBoatInfo] = useState<any>({});
+  const [loading, setLoading] = useState<boolean>(true);
+  const [data, setData] = useState<any>({});
```

Comment:
Same comments

boat info is type boat
data here is the type `Fuel` (May neet to be added)

Actually here we don't load the boat info or the data info as it is a new record.